doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1747,69 +1718,67 @@ name of the variable (here :g:`n`) is chosen based on :g:`T`.
              name the variable in the current goal and in the context of the
              existential variable. This can lead to surprising behaviors.
 
-.. tacv:: instantiate (@natural := @term)
-
-   This variant selects an existential variable by its position.  The
+   The second form refines an existential variable selected by its position.  The
    :n:`@natural` argument is the position of the existential variable
-   *from right to left* in the conclusion of the goal.  (Use one of
+   *from right to left* in the goal.  (Use one of
    the variants below to select an existential variable in a
    hypothesis.)  Counting starts at 1 and multiple occurrences of the
    same existential variable are counted multiple times.  Because this
    variant is not robust to slight changes in the goal, its use is
    strongly discouraged.
 
-.. tacv:: instantiate ( @natural := @term ) in @ident
-          instantiate ( @natural := @term ) in ( value of @ident )
-          instantiate ( @natural := @term ) in ( type of @ident )
+   :n:`in @ident`
+     Selects the hypothesis :n:`@ident`.
 
-   These allow to refer respectively to existential variables occurring in a
-   hypothesis or in the body or the type of a local definition (named :n:`@ident`).
+   :n:`in %|- *`
+     Selects all hypotheses and the goal.
 
-.. tacv:: instantiate
+   .. todo PR above is correct (in |- *)?
```

Comment:
If this is consistent with the occurrence clauses documented above, this should be only in the goal, so equivalent to not provided.

Reply:
You mean the description is correct and I can remove the note?

Reply:
No, I mean the description should not talk about hypotheses, only the goal. And it could mention that it is the default behavior.